src/Program.cs

```diff
@@ -8,6 +8,7 @@
 using AzureMcp.Models;
 using AzureMcp.Models.Command;
 using AzureMcp.Services.Azure.AppConfig;
+using AzureMcp.Services.Azure.AzureISV.Datadog;
```

Comment:
AzureISV should be AzureIsv. Any acronym 3 letters or longer in .NET public surface area should only have its first letter capitalized (eg Json, Http, Aspnet, AzureMcp, etc.). This is part of the .NET Design Guidelines.